docs/user/private-app-repository.md

```diff
@@ -101,6 +101,14 @@ To add Harbor as the private chart repository, go to `Configuration > App Reposi
 
 Once you create the repository you can click on the link for the specific repository and you will be able to deploy your own applications using Kubeapps.
 
+### Harbor: Authentication/Authorization
+
+It is possible to configure Harbor to use authentication:
+
+- Using HTTP [basic authentication](https://github.com/goharbor/harbor/blob/master/docs/user_guide.md#working-with-helm-cli). To use this feature:
+  - When creating a new project for serving as the helm chart repository in Harbor, set the `Access Level` of the project to non public. This enforces authentication to access the charts in the chart repository via Helm CLI or other clients.
```

Comment:
can you add a screenshot for this?

Reply:
@andresmgot The screenshot is https://raw.githubusercontent.com/kubeapps/kubeapps/master/docs/img/harbor-new-project.png (attached in the above step for creating new project in Harbor) which shows 'public access'. Uncheck 'Public' means non public access.  We can attach another snapshot with 'Public' unchecked, but it seems tedious. Any thought?

Reply:
I see, from the wording it seemed a different thing for me, it's okay then.